src/detect-modbus.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 ANSSI
+ * Copyright (C) 2014-2022 ANSSI
```

Comment:
no need to update this here, as we're only removing code

Reply:
Ok. Noted. Thank you.